var/spack/repos/builtin/packages/dyninst/package.py

```diff
@@ -40,6 +40,7 @@ class Dyninst(Package):
     depends_on("libdwarf", when='@:9')
     depends_on("boost@1.42:")
     depends_on('libiberty+pic')
+    depends_on("tbb@2018.6:")
```

Comment:
you may want to add when='@develop' for the tbb dependency, since older versions do not depend on tbb.